x-pack/plugins/security_solution/public/timelines/components/timeline/footer/index.tsx

```diff
@@ -197,12 +204,14 @@ export const PagingControlComponent: React.FC<PagingControlProps> = ({
   }
 
   return (
-    <EuiPagination
-      data-test-subj="timeline-pagination"
-      pageCount={totalPages}
-      activePage={activePage}
-      onPageClick={onPageClick}
-    />
+    <TimelinePaginationContainer hideLastPage={totalPages > 9999}>
```

Comment:
The `totalPages > 9999` check is evaluating to `false` in the gif below because the intent of this condition is to hide the last page when the estimated `totalHits`, in this case `113,248`, is greater than the estimation threshold `9999`, but with a page size of `100` (in the gif below), `totalPages` is `1133` (`113248 / 100 = 1132.38`)

As a result, the query fails when the (estimated) last page is clicked, per the gif below:

![totalPages](https://user-images.githubusercontent.com/4459398/95142423-e2cf6780-0730-11eb-81db-73bb55514fd9.gif)

This comparison will work correctly if it uses `totalHits` instead of `totalPages`.At present, `PagingControlProps` doesn't have a `totalHits` property. This may be addressed in a follow-up PR.